### PR TITLE
proteinortho 5.11 (new formula)

### PR DESCRIPTION
**Pull request by tseemann**
Proteinortho is a tool to detect orthologous genes within different species. For doing so, it compares similarities of given gene sequences and clusters them to find significant groups. The algorithm was designed to handle large-scale data and can be applied to hundreds of species at one.

### CODE REVIEW

**Comment by tseemann** [comment]
That's odd. I used "brew audit proteinortho.rb --strict" and fixed everything. Does the autobuild do something extra? Or is linuxbrew behind?

**Comment by iml** [comment]
`audit --strict` only runs on OS X >= Mavericks.

File: proteinortho.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  # tag "bioinformatics"
+  # doi "10.1186/1471-2105-12-124"
+
+  depends_on "Thread::Queue" => :perl
+  depends_on "File::Basename" => :perl
+
+  def install
+    rm "proteinortho5_clean_edges"
+    rm "proteinortho5_clustering"
+    # system "make", "INSTALLDIR=#{bin}", "install"
```
Comment (sjackman) [inline proteinortho.rb]: Please remove commented-out code, or explain with a comment why it's commented out.

**Comment by sjackman** [comment]
Please squash the three commits to a single commit. `git rebase -i HEAD~3` is useful for this.

**Comment by tseemann** [comment]
@sjackman Looks good to go!

**Comment by sjackman** [comment]
@BrewTestBot test this please

**Comment by tseemann** [comment]
Fixed typo in 'assert'.... @BrewTestBot please test this my old frenemy.

**Comment by sjackman** [comment]
The Mavericks failure appears to be transient.

```
==> brew pull --clean https://github.com/Homebrew/homebrew-science/pull/1955 FAILED
fatal: ref HEAD is not a symbolic ref
==> Applying patch
Applying: proteinortho 5.11 (new formula)
Using index info to reconstruct a base tree...
Falling back to patching base and 3-way merge...
Auto-merging proteinortho.rb
CONFLICT (add/add): Merge conflict in proteinortho.rb
Failed to merge in the changes.
Patch failed at 0001 proteinortho 5.11 (new formula)
The copy of the patch that failed is found in:
   /usr/local/Library/Taps/Homebrew/homebrew-science/.git/rebase-apply/patch
When you have resolved this problem, run "git am --continue".
If you prefer to skip this patch, run "git am --skip" instead.
To restore the original branch and stop patching, run "git am --abort".
Error: Patch failed to apply: aborted.
```

**Comment by sjackman** [comment]
Thanks, Torsten.

**Pull request closed without merging**